scripts/zones/Bostaunieux_Oubliette/npcs/Novalmauge.lua

```diff
@@ -15,39 +15,38 @@ require("scripts/globals/quests")
 
 local path =
 {
-    41.169430, -24.000000, 19.860674,
-    42.256676, -24.000000, 19.885197,
-    41.168694, -24.000000, 19.904638,
-    21.859211, -24.010996, 19.792259,
-    51.917370, -23.924366, 19.970068,
-    74.570229, -24.024828, 20.103880,
-    44.533886, -23.947662, 19.926519
+    [1] = {20.000, -24.000, 20.000, 0},
+    [2] = {50.000, -24.000, 20.000, 0},
+    [3] = {75.000, -24.000, 20.000, 0},
+    [2] = {50.000, -24.000, 20.000, 0}
```

Comment:
123..2?

In either case, the table keys aren't needed; Lua will internally give un-keyed elements an increasing integer representation 😉 

(Unless you've changed the key structure with hard-indexed elements, but that's a different can of worms.)

Reply:
yeah true, i guess i got so used to doing table like this for readability, that i just did it this way.

Reply:
all tables that use the new system should now be a simple point i.e. {10, 10, 10} with additional extra params able to be passed if required.